Main user guide build warnings (and they weren't caught by actions)

**Describe the bug**
Warnings generated when building user's guide

**To Reproduce**
I did standard setup, and `make SIMULATION=native prep` but likely exists for any config.
Then `make usersguide`, observe warnings in build/doc/warnings.log:
```
/export/home/jhageman/cFS/cFS-GitHub/osal/src/os/inc/osapi-bsp.h:95: warning: end of file while inside a group
/export/home/jhageman/cFS/cFS-GitHub/osal/src/os/inc/osapi-idmap.h:274: warning: end of file while inside a group
```

**Expected behavior**
No warnings.
Also the new GitHub actions should have caught this.  @astrogeco 

Note the warnings file at https://github.com/nasa/cFS/suites/1693812098/artifacts/31817568 is not empty... not clear why this didn't trigger a failure.

**Code snips**
None yet.

**System observed on:**
 - Hardware: cFS Dev Server
 - OS: Ubuntu 18.04
 - Versions: Bundle main

**Additional context**
N/A

**Reporter Info**
Jacob Hageman - NASA/GSFC

--- a/.github/workflows/build-documentation.yml
+++ b/.github/workflows/build-documentation.yml
@@ -103,8 +103,8 @@ jobs:
 
       - name: Warning Check
         run: |
-          if [[ -s build/doc/warnings.log ]]; then
-            cat build/doc/warnings.log
+          if [[ -s usersguide_warnings.log ]]; then
+            cat usersguide_warnings.log
             exit -1
           fi
 
@@ -159,7 +159,7 @@ jobs:
 
       - name: Warning Check
         run: |
-          if [[ -s build/doc/warnings.log ]]; then
-            cat build/doc/warnings.log
+          if [[ -s osalguide_warnings.log ]]; then
+            cat osalguide_warnings.log
             exit -1
           fi

